doc/user/content/manage/access-control/_index.md

```diff
@@ -1,119 +1,139 @@
+help prevent unauthorized or improper access to sensitive objects. For example,
+organization administrators can sue RBAC to:
 
-### Inviting users
+* Determine which users have read or write privileges for specific objects.
 
-As an **administrator**, you can invite new users via the Materialize Console.
-Depending on the level of access each user should have, you can assign them
-`Organization Admin` or `Organization Member` privileges.
+* Control how users interact with clusters by giving them different levels of
+access to resources.
 
-   - `Organization Admin`: can perform adminstration tasks in the console, like
-     inviting new users, editing account and security information, or managing
-     billing. Admins have _superuser_ privileges in the database.
+* Prevent accidental operations from unauthorized users.
 
-   - `Organization Member`: can log in to the console and has restricted access
-     to the database, depending  on the privileges defined via
-     [role-based access control (RBAC)](#role-based-access-control-rbac).
+* Isolate access to user-facing data from internal organization data.
 
-These privileges primarily determine a user's **access level to the console**,
-but also have implications in the **default access level to the database**.
-More granular permissions within the database must be handled separately using
-[role-based access control (RBAC)](#role-based-access-control-rbac).
+A role is a collection of privileges you can apply to users. Roles make it
+easier to assign or revoke privileges on Materialize objects. You can group
+users into specified roles with different levels of privileges and adjust those
+privileges to ensure they have the correct level of access to objects.
 
-[//]: # "TODO(morsapaes) Add a specific anotation for tutorial call-outs, to
-make these more noticeable."
+In Materialize, you have both:
 
-To invite users to your Materialize organization, follow [this step-by-step guide](/manage/access-control/invite-users).
+- Predefined [Organization roles](#organization-roles)
 
-### Configuring single sign-on (SSO)
+- Custom user roles (associated with a specific user/function/etc.)
 
-As an **administrator** of a Materialize organization, you can configure single
-sign-on (SSO) as an additional layer of account security using your existing
-[SAML](https://auth0.com/blog/how-saml-authentication-works/)- or
-[OpenID Connect](https://auth0.com/intro-to-iam/what-is-openid-connect-oidc)-based
-identity provider. This ensures that all users can securely log in to the
-Materialize Console using the same authentication scheme and credentials across
-all systems in your organization.
+## Organization roles
 
-To configure SSO for your Materialize organization, follow [this step-by-step guide](/manage/access-control/sso).
+Each user/service account in Materialize is associated with an organization
+role:
 
-### Configuring network policies
```

Comment:
Just moved the configure network policies tutorial out from under manage-access (and this intro is already repeated in the tutorial)
https://preview.materialize.com/materialize/33102/manage/manage-network-policies/